express.js

```diff
@@ -149,7 +149,16 @@ app.post('/feedback', (req, res) => {
     }];
   }
 
-  const webhook = new WebhookClient({ url: hookURL });
+  let webhook;
+  try {
+    webhook = new WebhookClient({ url: hookURL });
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      message: 'An error occured on the server. If you are a developer, your webhook url is likely wrong.'
+    });
```

Comment:
Needs a `return` after `res.status()` so that it doesn't continue on to the `webhook.send()`

Reply:
Should be resolved now! Sorry for the long delay